### PR TITLE
Fix docker setup

### PR DESCRIPTION
**Pull request by roschaefer**
This PR has commits of #147 and should be rebased after #147 is merged.

- [x] expose minimal number of ports in `Dockerfile`

### CODE REVIEW

**Comment by roschaefer** [comment]
I would like to test `thumbor` in the docker environment. The service *is up and running* but unfortunately I cannot see anything. Do thumbnails get created in development?

**Comment by appinteractive** [comment]
If you want to use thumbor in docker you have to add the key and url to the config. Currently the url is empty that’s why it fails

**Comment by roschaefer** [comment]
@appinteractive that's the README, I've set the url. Should a database seeding in development environment create thumbnails after all?

**Comment by appinteractive** [comment]
@roschaefer yes the seeding in development gets random image urls which are downloaded by the remote-images hook and then returned as (secure) thumbor urls (if configured, otherwise local paths) which then gets served by thumbor (if its running)

File: Dockerfile [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
 # expose the app port
 EXPOSE 3030
 
-# set environment variables
-# ENV NPM_CONFIG_PRODUCTION=false
-# ENV HOST=0.0.0.0
-ENV NODE_ENV=production
-ENV API_PORT=3030
+# create working directory
+RUN mkdir -p /API
```
Comment (Mastercuber) [inline Dockerfile]: Ich würde das COPY . /API direkt nach dem erstellen des API Verzeichnises machen, dann muss man später die package.json und yarn.lock auch nicht reinkopieren.

File: Dockerfile [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
-RUN mkdir -p /var/www/
-WORKDIR /var/www/
+RUN yarn global add pm2
 
 # install app dependencies
-COPY package.json /var/www/
-COPY yarn.lock /var/www/
+COPY package.json /API
+COPY yarn.lock /API
 RUN yarn install --frozen-lockfile --non-interactive
```
Comment (Mastercuber) [inline Dockerfile]: Nachdem die Abhängigkeiten installiert wurden musste ich noch `RUN yarn global add cross-env nodemon` ausführen, weil das starten der API und mongod sonst nicht ging. Gerade startet bei mir nur die API nicht. Es kann `server-favicon` nicht geladen werden..

Comment (roschaefer) [reply Dockerfile]: I removed all my images, volumes and containers from the cache and rebuild everything and I cannot reproduce that error. Super weird.

**Comment by roschaefer** [comment]
@appinteractive Why is codacy complaining about a missing maintaner? The `MAINTAINER` directive is deprecated.

**Comment by appinteractive** [comment]
@roschaefer we don’t have to bother about that codacy issue for now.

**Pull request merged**